internal/node/test/BUILD.bazel

```diff
@@ -604,3 +604,75 @@ nodejs_test(
         "@bazel_tools//src/conditions:windows": "@node16_windows_amd64//:node_toolchain",
     }),
 )
+
+# these rules create different targets for each toolchain that is supplied in the list for toolchains
+# this makes it easy to build or test against multiple versions of a toolchain at the same time with less duplicate work
+
+# this will create 2 nodejs_binary targets
+# bazel build //internal/node/test:main_multiple_0
+# bazel build //internal/node/test:main_multiple_1
+# the numbers are based on the order that the toolchains are passed to the rule
+nodejs_binary_toolchains(
```

Comment:
it might even be better to show it with a comprehension rather than a macro

```
[
  nodejs_binary(name = "x" + tc, ...)
  for tc in [...]
]
```

Reply:
So this would be in the case we decide to remove the macro mentioned in the above comment?
Is that the preferred outcome?

Reply:
We want to keep our maintenance burden in the upstream repo minimal, since it's not really funded. So any code maintained here ought to be broadly applicable. If only a couple users need it, then my instinct is it should live in their repo (we can have an example to help them).

Reply:
Sounds Good! thanks for the clarification